daml-lf/engine/src/main/scala/com/digitalasset/daml/lf/engine/EngineConfig.scala

```diff
@@ -35,9 +35,8 @@ final case class EngineConfig(
     stackTraceMode: Boolean = false,
     profileDir: Option[Path] = None,
     contractKeyUniqueness: ContractKeyUniquenessMode = ContractKeyUniquenessMode.On,
-    // TODO: https://github.com/digital-asset/daml/issues/10504
-    //  switch the default to true
-    requireSuffixedGlobalCids: Boolean = false,
+    requireV1ContractId: Boolean = false,
```

Comment:
maybe forbidV0 would be more future proof if we ever add v2?

Reply:
:+1:

Reply:
Ok so in canton, I would rename use use of `requireV1ContractId` to `forbidV0ContractId`.

And no need to use `requireV1ContractIdSuffix`, right, @remyhaemmerle-da ?

Reply:
@oliverse-da you should use both: `forbidV0ContractId` to make sure you never get a global v0 cid. `requireV1ContractIdSuffix` to make sure that all global v1 cids have a suffix.